### PR TITLE
Add profiler output directory to .gitignore

### PR DESCRIPTION
**Pull request by dszmigielski**
There was change in DataDog repo that causes running 'docker-compose run Profiler' to create output files that are not ignored by git. Link to DataDog PR: https://github.com/DataDog/dd-trace-dotnet/commit/0c3d8e5591c59dcffe3d334e48dc14ff05da9ef9 
The output path in build/docker/Datadog.Trace.ClrProfiler.Native.sh was changed from 'obj/Debug/x64' to 'build', the first one was obviously caught by .gitignore, the later is not, causing number of files to be added when running profiler build.

### CODE REVIEW

**Comment by pellared** [comment]
@zacharycmontoya @kevingosse @colin-higgins @tonyredondo 
You may want to review it and cherry-pick this commit into https://github.com/DataDog/dd-trace-dotnet

**Review by pjanotti — APPROVED** [review]
This LGTM but I would like to hear from @zacharycmontoya, @tonyredondo, or @macrogreg before merging.

File: .gitignore [hunk 1]
```diff
@@ -274,4 +274,7 @@ deploy/AzureAppServices/
 # macOS temporal files
 *..DS_Store
 .DS_Store/*
-.DS_Store
\ No newline at end of file
+.DS_Store
```
Comment (pellared) [inline .gitignore]: AFAIK just `.DS_Store` should be enough

**Review by pellared — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (pellared) [inline .gitignore]: AFAIK just `.DS_Store` should be enough

Comment (pellared) [inline .gitignore]: AFAIK just `.DS_Store` should be enough

**Pull request merged**